### DISCUSSION TITLE
AR Scale Problem / No udsz file needed anymore?

### DISCUSSION THREAD

**Discussion by Aztec91** [google/model-viewer#2892] [General]
Hey guys,

long time no see here.

I have a question:

https://commacross.de/3DAR/ar3.html 

If you acess the site with your phone, the model ist scaled way too hughe in AR.
I tried to scale it down the smalles I can in blender, but still has no effect it seems.

Any Idea how I can fix that? the scale-function didnt work out for me.

Also is it true, that you no longer need a seperate file for iOs Devices? That is super cool, thank you so much.

Kind regards
Aztec

**Comment by samaneh-kazemi** [comment]
Hi,

@elalish Any thoughts for the scale question?

And that's correct as of v1.7.☺️

https://modelviewer.dev/docs/index.html#entrydocs-augmentedreality-attributes-arModes


https://github.com/google/model-viewer/releases/tag/v1.7.0

**Reply by elalish** [reply]
Scale problems come down to units; glTF is always defined in meters, so the common problem is to model something  at centimeters scale and have it show up 100x too large. You can use [getDimensions()](https://modelviewer.dev/docs/#entrydocs-loading-methods-getDimensions) to see how large it is in meters, and you can fix it  with [scale](https://modelviewer.dev/docs/#entrydocs-scenegraph-attributes-scale).
